packages/vuetify/src/util/helpers.ts

```diff
@@ -150,6 +150,28 @@ export function addOnceEventListener (el: EventTarget, event: string, cb: () =>
   el.addEventListener(event, once, false)
 }
 
+let passiveSupported = false
+try {
+  const testListenerOpts = Object.defineProperty({}, 'passive', {
+    get: () => {
+      passiveSupported = true
+    }
+  })
+
+  window.addEventListener('testListener', () => {}, testListenerOpts)
+  window.removeEventListener('testListener', () => {}, testListenerOpts)
```

Comment:
This doesn't do anything. The listener remains in the DOM.

Reply:
Doh. Anonymous functions... Fixing this now.